restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/util/RCUtils.java

```diff
@@ -173,4 +177,90 @@ public static void validateConnectionParms(HashMap<String, Object> parameters) t
       }
 
    }
+
+   public static boolean shouldUpdatePush(HashMap<String, Object> parameters) throws RCException{
+      if (!parameters.containsKey(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_FCM_SERVER_KEY) ||
```

Comment:
I noticed that even when I change settings in an irrelevant value like ICE Candidate Timeout, we still registerForPush(). Can we avoid and make this method return false if none of the push notification related parameters have changed? That way we make sure that we ask server to update only if any of the push related parameters have changed. Wdyt?